src/Servers/HttpSys/src/RequestProcessing/RequestContext.FeatureCollection.cs

```diff
@@ -36,6 +36,7 @@ internal partial class RequestContext :
     IHttpResponseTrailersFeature,
     IHttpResetFeature,
     IHttpSysRequestDelegationFeature,
+    IHttpSysRequestPropertyFeature,
```

Comment:
Smoke test that we expose this feature on the HttpContext?
e.g. https://github.com/dotnet/aspnetcore/blob/3628f0b643accd0cc9e57fac0373964928197544/src/Servers/HttpSys/test/FunctionalTests/HttpsTests.cs#L251

Reply:
added